terraform_files/main.tf

```diff
@@ -36,6 +36,13 @@ resource "libvirt_network" "net" {
       ip = var.api_vip
       hostname = "api.${var.cluster_name}.${var.cluster_domain}"
     }
+    dynamic "hosts" {
+      for_each = var.extra_dns_names
+      content {
+        hostname = hosts.key
+        ip = hosts.value
+      }
+    }
```

Comment:
Is it still needed when you use the IP address as NTP source?

Reply:
No